main.html

```diff
@@ -57,7 +57,9 @@
 <script type="text/javascript" src="./js/serial_backend.js"></script>
 <script type="text/javascript" src="./js/data_storage.js"></script>
 <script type="text/javascript" src="./js/fc.js"></script>
-<script type="text/javascript" src="./js/msp.js"></script>
+<script type="text/javascript" src="./js/msp/MSP_codes.js"></script>
```

Comment:
Your capitalisation is inconsistent. Also, I would like to get away from using snake case in favour of using camel case, particularly in cases where the object defined in a file is camel case already.